modules/apps/headless/headless-object/headless-object-impl/rest-openapi.yaml

```diff
@@ -62,6 +81,44 @@ components:
                 -   actionIds
                 -   type
             type: object
+        ObjectEntryCMSBulkActionRequest:
+            description:
+                Represents the request for CMS bulk actions
+            properties:
+                ids:
```

Comment:
I am worried about getting just a list of ids, as in the code, I saw you are calling the fetch methods to check if there is an Object Entry with that id, an Object Folder, etc. If this endpoint is called with 20 possible types of elements (such as Account, User, Object Entry, Object Folder, etc), you will have to call the fetch methods up to 20 times. Maybe it is better to inform in the request the ids of each type of data